src/Materials/Node/nodeMaterialBlockConnectionPoint.ts

```diff
@@ -133,6 +133,11 @@ export class NodeMaterialConnectionPoint {
      */
     public name: string;
 
+    /**
+     * Gets or sets the connection point name
+     */
+    public displayName: string;

```

Comment:
You may want to serialize / deserialize it

Reply:
Added it to NodeMaterialBlockConnectionPoint.serialize()

Reply:
And what about deserialize???

Reply:
Working on that now. Is there already a deserialize method? I could not find one so I write a new one for NodeMaterialBlockConnectionPoint class.

Reply:
I added _deserializePortDisplayNames() to NodeMaterialBlock. I think it makes the most sense there but happy to discuss if otherwise.